lib/jmap/mail/calendar/reply/calendar_event_reply_response.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/core/method/response/event_action_response.dart';
+import 'package:jmap_dart_client/util/json_parsers.dart';
+
+class CalendarEventAcceptResponse extends EventActionResponse {
+  CalendarEventAcceptResponse(
+    super.accountId,
+    super.notFound,
+    {
+      this.accepted,
+      this.notCreated
+    });
+
+  final List<Id>? accepted;
+  final List<Id>? notCreated;
+
+  @override
+  bool get eventActionSuccess => super.eventActionSuccess
+    && (accepted != null && accepted!.isNotEmpty)
+    && (notCreated == null || notCreated!.isEmpty);
```

Comment:
IMO, it's unnecessary, because it's not clear that in some cases there are both success and failure events

Reply:
Removed